tests/test_model_estimator_compatibility.py

```
@@ -13,19 +13,16 @@ def test_fit_config_model_estimator_compatibility_matrix(
     model: str, estimator: str
 ) -> None:
     """Every advertised model-estimator pair must match executable fit support."""
-    config = FitConfig(model=model, estimator=estimator)
-
     if model == "BIFAC2PLM" and estimator == "jmle":
         with pytest.raises(ValueError, match="BIFAC2PLM.*mmle"):
```

Ruff (0.16.1)

[warning] Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)